public/app/plugins/panel/dashlist/DashList.tsx

```diff
@@ -140,27 +148,14 @@ export function DashList(props: PanelProps<Options>) {
   ];
 
   const css = useStyles2(getStyles);
+  const [urlParams, onReInterpolateUrlMacros] = useDashListUrlParams(props);
 
   const renderList = (dashboards: Dashboard[]) => (
-    <ul>
+    <ul onMouseEnter={onReInterpolateUrlMacros} onFocus={onReInterpolateUrlMacros}>
```

Comment:
:/ I don't know. I understand both solutions are far from ideal, and this actually fixes it for Scenes, but reinterpolating on mouseover feels _much_ worst of a solution than listening for the global event.

_Personally_, I'm okay with this not working in Scenes until we get a better longer term solution, but I'm not sure what that would involve. Happy to leave the final say up to others.

Reply:
@joshhunt yea, agree. just dislike the VariablesChanged event, the event payload does not make sense.  it is emitted on the global appEvents and not the PanelContext.eventBus, etc. 

But I think it's a still better solution than onMouseEnter/onFocus., updated the solution back to using it and now emitting this event from DashboardScene when scene variables change.